.github/CODEOWNERS

```diff
@@ -1 +1,9 @@
 * @sfc-gh-turbaszek @sfc-gh-pjob @sfc-gh-jsikorski @sfc-gh-astus
+
+# Native Apps Owners
+src/snowcli/cli/nativeapp/ @sfc-gh-bgoel @sfc-gh-cgorrie @sfc-gh-bdufour @sfc-gh-melnacouzi
```

Comment:
There is no NADE Github group in the Snowflake-Labs org, and since moving to a different org is an upcoming goal of this repo, I was advised to add names individually. Once it is moved to the final org, this file will be revisited to add NADE as a team instead of individual members.